### DISCUSSION TITLE
display images in README.md

### DISCUSSION THREAD

**Discussion by yokomeow** [treeverse/lakeFS#5357] [Q&A] [answered]
Is it possible to paste images **from lakefs repo** in README.md from the same repo using markdown?
If yes, how to write an image path correctly?

**Comment by ozkatz** [comment]
I believe it is! Tagging @eladlachmi which I believe implemented something like this recently?

**Reply by eladlachmi** [reply, marked as answer]
Yes, this has been implemented.
